.github/workflows/changelog.yml

```diff
@@ -0,0 +1,12 @@
+name: Check changelog file included
+
+on:
+  pull_request:
+    paths:
+      - 'panels/**'
```

Comment:
I assume we don't need the following folders present in the root directory of the repository
- `chaoss-gmd-cde`
- `dashboards-cauldron`
- `src`

Reply:
I just ignored them because they are not included in the release. I'm not sure if those directories are there for some reason, but as far as I know they are not used.

Reply:
Cool, makes sense to me. Just want to confirm once again.
Thanks @jjmerchante for replying.